runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowPipelineRegistrar.java

```diff
@@ -52,7 +52,7 @@ private DataflowPipelineRegistrar() { }
     @Override
     public Iterable<Class<? extends PipelineRunner<?>>> getPipelineRunners() {
       return ImmutableList.<Class<? extends PipelineRunner<?>>>of(
-          DataflowRunner.class);
+          DataflowRunner.class, TestDataflowRunner.class);
```

Comment:
I think we were intentionally keeping this out of the registrar so it didn't confuse users.